Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnUpdate.hs

```diff
@@ -149,12 +153,20 @@ buildOnUpdateMessage req@RideCompletedBuildReq {} = do
       OnUpdate.RideCompleted
         RideCompletedOU.RideCompletedEvent
           { id = ride.bookingId.getId,
-            update_target = "fulfillment.state.code,quote.price,quote.breakup",
+            update_target = "fulfillment.state.code,quote.price,quote.breakup,payment.uri",
```

Comment:
Can you make it backward compatible?

Reply:
@hkmangla content o this field never used on bap side
So these changes backward compatible itself